apheleia.el

```diff
@@ -922,6 +922,7 @@ being run, for diagnostic purposes."
     (ocamlformat . ("ocamlformat" "-" "--name" filepath
                     "--enable-outside-detected-project"))
     (prettier . (npx "prettier" "--stdin-filepath" filepath))
+    (shfmt . ("shfmt" "-ci"))
```

Comment:
Should we use the default settings by default, and let people customize it if they want to add additional options? Or do you think most people would want `-ci` so we should include it by default?

Reply:
Yeah you're right here, we shouldn't make decisions by default.